README.md

```diff
@@ -45,6 +45,17 @@ The Data Management panel provides the following capabilities:
 
 Note: the sample data provided here is basic so users are encouraged to look also at the Demo project where additional and more extensive data loading capabilities are available.
 
+## === Bootstrapping Ecoverse roles ===
+
+In order to create Ecoverse profiles, do the following:
+
+1. Set AUTHENTICATION_ENABLED=false environment variable
+1. If you are running an old version of the application (<0.3.*) do a Reset Ecoverse + Sample Data from the Data Management Panel.
+1. Prepare a bootstrap json. Default json file is provided in src/utils/config/authorisation-bootstrap.json. If you'd like to change the name / location of the bootstrap file, set AUTH_BOOTSTRAP_PATH environment variable with the full path to your file.
```

Comment:
Looks like it is searching from the js file, so for me:
Error: Cannot find module 'C:\Users\neil\Documents\Dev_CT\Server\dist\utils\config\authorisation-bootstrap.json'

Reply:
Can't reproduce that. Can you share more details?